src/GraphQLParser.Tests/ParserTests.cs

```diff
@@ -527,6 +527,18 @@ public void Should_Throw_On_Unknown_OperationType()
             ex.Column.ShouldBe(1);
         }
 
+        [Theory]
+        [InlineData("enum E { true A }", "Unexpected Name \"true\"; enum values are represented as unquoted names but not 'true' or 'false' or 'null'.", 1, 10)]
+        [InlineData("enum E { B false }", "Unexpected Name \"false\"; enum values are represented as unquoted names but not 'true' or 'false' or 'null'.", 1, 12)]
+        [InlineData("enum E { A null B }", "Unexpected Name \"null\"; enum values are represented as unquoted names but not 'true' or 'false' or 'null'.", 1, 12)]
+        public void Should_Throw_On_Invalid_EnumValue(string query, string description, int line, int column)
```

Comment:
Test failed before changes since EnumValueDefinition accepts true/false/null that contradicts spec.